src/main/java/org/broadinstitute/hellbender/tools/exome/FilterTargets.java

```diff
@@ -94,6 +94,9 @@
     public static final String REJECT_OUTPUT_FILE_FULL_NAME = "rejectedOutputFile";
     public static final String REJECT_OUTPUT_FILE_SHORT_NAME = "rejected";
 
+    public static final String EXCLUDED_INTERVAL_FULL_NAME = "excludeInterval";
```

Comment:
Slight inconsistency in names -- there is the constant `EXCLUDED_INTERVAL`, the string literal `"excludeInterval"` and the variable `excludeIntervals`.  My vote for all is `excludedIntervals` and I will comment wherever relevant.